kingpin/actors/aws/cloudformation.py

```diff
@@ -824,11 +842,14 @@ def __init__(self, *args, **kwargs):
         # Check if the supplied CF template is a local file. If it is, read it
```

Comment:
I think we should raise an exception if `template` and `template_url` are supplied.. rather than just defaulting to one or the other silently.